docs/changes.md

```diff
@@ -1,5 +1,6 @@
 ## Changes
 
+- [`4.1`](https://github.com/ibm-mas/cli/releases/tag/4.1.0) Added new Manage core functional tests and separate results from Industry Solutions by using different product ids (#241)
```

Comment:
I think 4.1 is already out (and missed in the changelogs), this would be 4.2

Reply:
@whitfiea done. I have also added the missing 4.1 item [there](https://github.com/ibm-mas/cli/pull/241/files#diff-20bcbf4917b5594b6151452760fa22e23c2ffd2f33683329e1f602b9164e31e5R3) (@durera fyi)